### DISCUSSION TITLE
Welcome to docx4j Discussions!

### DISCUSSION THREAD

**Discussion by plutext** [plutext/docx4j#426] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. 

  You can post questions here, on StackOverflow, or in the docx4j forums.  If a question morphs into a bug, it can be converted to an issue.

  We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with and help other community members.

**Comment by swilliams7996** [comment]
Hi, I'm trying to figure out how to replace variables in header for word. I'm able to do that in the body of the file but can't do it in the header of a word file. Can someone give me an example of how to do this?
